### PR TITLE
Don’t include timestamp/connectionid in string/json representations of messages

### PR DESCRIPTION
**Pull request by SimonWoolf**
And only include clientId in json rep if present

Fixes https://github.com/ably/ably-js/issues/44

### CODE REVIEW

File: common/lib/types/message.js [hunk 1]
```diff
@@ -18,12 +18,12 @@ var Message = (function() {
 	Message.prototype.toJSON = function() {
 		var result = {
 			name: this.name,
-			clientId: this.clientId,
-			connectionId: this.connectionId,
```
Comment (paddybyers) [inline common/lib/types/message.js]: It is valid to include a `connectionId` in `Message` when sent over REST so I don't think it's valid to remove it here.

File: common/lib/types/presencemessage.js [hunk 2]
```diff
@@ -50,8 +50,6 @@ var PresenceMessage = (function() {
 		result += '; action=' + this.action;
 		if(this.id)
 			result += '; id=' + this.id;
-		if(this.timestamp)
```
Comment (paddybyers) [inline common/lib/types/presencemessage.js]: Why is this removed?

File: common/lib/types/message.js [hunk 3]
```diff
@@ -43,12 +43,8 @@ var Message = (function() {
 			result += '; name=' + this.name;
 		if(this.id)
 			result += '; id=' + this.id;
-		if(this.timestamp)
```
Comment (paddybyers) [inline common/lib/types/message.js]: Why is this removed (also `connectionId` below) ?

**Comment by SimonWoolf** [comment]
@paddybyers sorry, I misread the issue asking for timestamp/connectionId to be removed from toString as well as toJSON

**Comment by paddybyers** [comment]
Thanks
